spec/requirements_spec.rb

```diff
@@ -0,0 +1,183 @@
+require 'rails_helper'
+
+RSpec.describe 'Requirements with without user logged in', type: :system do
+  context 'splash page' do
+    it 'should diplay a splash page if user not logged in' do
+      visit groups_path
+      expect(page).to have_selector('.splash')
+    end
+    it 'should display a link to the login page if user not logged in' do
+      visit groups_path
+      expect(page).to have_link('LOG IN')
+    end
+    it 'should display a link to the sign up page if user not logged in' do
+      visit groups_path
+      expect(page).to have_link('SIGN UP')
+    end
+    it 'user should not be able to see the groups page if user not logged in' do
+      visit groups_path
+      expect(page).to_not have_selector('.groups')
+    end
+  end
+end
+
+RSpec.describe 'Requirements with user logged in', type: :system do
+  context 'sign up and log in pages' do
+    it 'user should be able to register and login into the app using email and password' do
+      visit new_user_registration_path
+      name = Faker::Name.name
+      email = Faker::Internet.email
+      password = Faker::Internet.password
+      @test_user = build(:user, name:, email:, password:, password_confirmation: password)
+      @test_user.skip_confirmation!
+      fill_in 'Name', with: name
+      fill_in 'Email', with: email
+      fill_in 'Password', with: password
```

Comment:
- [ ] You have done well writing tests for this project 👏  However, this test is failing. Kindly fix it so it passes 👍 
![image](https://user-images.githubusercontent.com/46839548/182971854-15236cfa-2030-43e1-985d-6c064814f4a5.png)

Reply:
Will do 👍